t/unit/utils/test_json.py

```diff
@@ -71,6 +70,11 @@ def test_UUID(self):
             lambda: uuid.uuid3(uuid.NAMESPACE_URL, "https://example.org"),
             uuid.uuid4,
             lambda: uuid.uuid5(uuid.NAMESPACE_URL, "https://example.org"),
+            # The uuids below correspond to v6, v7 and v8 respectively and were
+            # generated using the package uuid6.
+            lambda: uuid.UUID("1ee0b1e6-dd55-63d2-867f-88cb9205458f"),
```

Comment:
is there any need for doing like the old versions example or we are good to go?        
 lambda: uuid.uuid3(uuid.NAMESPACE_URL, "https://example.org"),
            uuid.uuid4,
            lambda: uuid.uuid5(uuid.NAMESPACE_URL, "https://example.org")